Sources/ReadingListKit/ReadingListModel.swift

```diff
@@ -0,0 +1,13 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+public class ReadingListModel {
+    let pocketAccessLayer: PocketAccessLayer
+
+    public init(authTokenProvider: @escaping () -> String, consumerKey: String) {
+        pocketAccessLayer = PocketAccessLayer(authTokenProvider, consumerKey)
+    }
+}
```

Comment:
curious to know (maybe with some comments) what this type will be responsible of.

Reply:
The Model responds to events from the View, it then determines how and where to get View-relevant data,  delegates data fetching and processes the responses into View Model for consumption by the view.

Reply:
> processes the responses into View Model

are you adopting `MVVM`?

Reply:
I wouldn't put a name on this, other than I like how it breaks up and encapsulates the work.